app/controllers/api/v1/projects_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class Api::V1::ProjectsController < ApplicationController
+  include JWTHelper
+
+  skip_before_action :verify_authenticity_token
+  before_action :authenticate_jwt
+
+  # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+  def create
+    populate_project_from_forge
+    @project.enlistments_attributes = { '0' => { 'code_location_attributes' => { 'scm_type' => 'git',
+                                                                                 'url' => params[:url],
+                                                                                 'branch' => params[:branch] } } }
+    @project.editor_account = current_user
+    @project.assign_editor_account_to_associations
+    @project.manages.new(account: current_user) if params[:managed_by_creator].to_bool
+    @project.save!
+    create_code_location_subscription if @project.enlistments.exists?
+  rescue StandardError => e
+    render json: "Bad params #{e}", status: :bad_request
+  end
+  # rubocop:enable Metrics/AbcSize, Metrics/MethodLength
+
+  private
+
+  def project_params
+    params.require(:project).permit(
+      :name, :description, :vanity_url, :url, :download_url, :managed_by_creator, :coverity_project_id,
+      project_licenses_attributes: [:license_id],
+      enlistments_attributes: [code_location_attributes: %i[branch url scm_type]]
+    )
```

Comment:
Project api will not be passing in params matching this hash. We won't have *enlistment_attributes*, *project_licenses_attributes* etc. Consider what the actual api call would look like.